k4GaudiPandora/options/runDDCaloDigi.py

```diff
@@ -0,0 +1,167 @@
+    calodigicol.CalibrHCALOther = [57.4588011802]
+    calodigicol.IfDigitalEcal = 0
+    calodigicol.MapsEcalCorrection = 0
+    calodigicol.IfDigitalHcal = 0
+    calodigicol.ECALGapCorrection = 1
+    calodigicol.ECALGapCorrection = 1
+    calodigicol.ECALEndcapCorrectionFactor = 1.03245503522
+    calodigicol.HCALEndcapCorrectionFactor = 1.000
+    calodigicol.ECALGapCorrectionFactor = 1.0
+    calodigicol.ECALModuleGapCorrectionFactor = 0.0
+    calodigicol.HCALModuleGapCorrectionFactor = 0.5
+
+    # timing parameters for ECAL
+    calodigicol.UseEcalTiming = 1
+    calodigicol.ECALCorrectTimesForPropagation = 1 
+    calodigicol.ECALTimeWindowMin = -1.0
+    calodigicol.ECALEndcapTimeWindowMax = 10.0
+    calodigicol.ECALBarrelTimeWindowMax = 10.0
+    calodigicol.ECALDeltaTimeHitResolution = 10.0
+    calodigicol.ECALTimeResolution = 10.0
+    calodigicol.ECALSimpleTimingCut = True
+
+    # timing parameters for HCAL
+    calodigicol.UseHcalTiming = 1
+    calodigicol.HCALCorrectTimesForPropagation = 1
+    calodigicol.HCALTimeWindowMin = -1.0
+    calodigicol.HCALEndcapTimeWindowMax = 10.0
+    calodigicol.HCALBarrelTimeWindowMax = 10.0
+    calodigicol.HCALDeltaTimeHitResolution = 10.0
+    calodigicol.HCALTimeResolution = 10.0
+    calodigicol.HCALSimpleTimingCut = True
+
+    # parameters for extra ECAL digitization effects
+    calodigicol.CalibECALMIP = 1.0e-4
+    calodigicol.ECALApplyRealisticDigi = 0
+    calodigicol.ECAL_PPD_PE_per_MIP = 7.0
+    calodigicol.ECAL_PPD_N_Pixels = 10000
+    calodigicol.ECAL_PPD_N_Pixels_uncertainty = 0.05
+    calodigicol.ECAL_miscalibration_uncorrel = 0.0
+    calodigicol.ECAL_miscalibration_uncorrel_memorise = False
+    calodigicol.ECAL_miscalibration_correl = 0.0
+    calodigicol.ECAL_deadCellRate = 0.0
+    calodigicol.ECAL_deadCell_memorise = False
+    calodigicol.ECAL_strip_absorbtionLength = 1.0e6
+    calodigicol.ECAL_pixel_spread = 0.05
+    calodigicol.ECAL_elec_noise_mips = 0.0
+    calodigicol.energyPerEHpair = 3.6
+    calodigicol.ECAL_maxDynamicRange_MIP = 2500.0
+    calodigicol.StripEcal_default_nVirtualCells = 9
+    calodigicol.ECAL_default_layerConfig = "000000000000000"
+
+    # parameters for extra HCAL digitization effects
+    calodigicol.CalibHCALMIP = 1.0e-4
+    calodigicol.HCALApplyRealisticDigi = 0
+    calodigicol.HCAL_PPD_PE_per_MIP = 10.0
+    calodigicol.HCAL_PPD_N_Pixels = 400
+    calodigicol.HCAL_PPD_N_Pixels_uncertainty = 0.05
+    calodigicol.HCAL_miscalibration_uncorrel = 0.0
+    calodigicol.HCAL_miscalibration_uncorrel_memorise = False
+    calodigicol.HCAL_miscalibration_correl = 0.0
+    calodigicol.HCAL_deadCellRate = 0.0
+    calodigicol.HCAL_deadCell_memorise = False 
+    calodigicol.HCAL_pixel_spread = 0.0
+    calodigicol.HCAL_elec_noise_mips = 0.0
+    calodigicol.HCAL_maxDynamicRange_MIP = 200.0
+
+
+#
+iosvc = IOSvc()
+iosvc.input = "../simulation/sim_partgun_1000.root"
+iosvc.output = "../outputfiles/DDCaloDigi/outputCaloDigi_Gaudi.root"
```

Comment:
```suggestion
iosvc.Input = "../simulation/sim_partgun_1000.root"
iosvc.Output = "../outputfiles/DDCaloDigi/outputCaloDigi_Gaudi.root"
```
It's giving a warning now if you use the lower caps version.